src/internal/m365/service/groups/backup.go

```diff
@@ -118,5 +129,49 @@ func ProduceBackupCollections(
 		collections = append(collections, baseCols...)
 	}
 
+	// Add metadata about sites
+	md, err := getSitesMetadataCollection(
+		creds.AzureTenantID,
+		bpc.ProtectedResource.ID(),
+		sitesPreviousPaths,
+		su)
+	if err != nil {
+		return nil, nil, false, err
+	}
+
+	collections = append(collections, md)
+
 	return collections, ssmb.ToReader(), canUsePreviousBackup, el.Failure()
 }
+
+func getSitesMetadataCollection(
+	tenantID, groupID string,
+	sites map[string]string,
+	su support.StatusUpdater,
+) (data.BackupCollection, error) {
+	// TODO(meain): Should we store this one level above? If so, how would we
```

Comment:
Let me know if anyone has any thoughts on the location/content for this metadata.

Reply:
I'm interpreting the question as, `"should we store the map of sites at the categoryType dir, instead of the sitePath-const dir?"`  If I read that correctly then, eh, it's sixes.  There's no difference in group resource (since one group holds many sites), and the siteIDs already compose unique keys.  But there's no harm in the namespacing.